src/components/TransactionType.tsx

```diff
@@ -7,51 +7,70 @@ import MultipleStopRoundedIcon from "@mui/icons-material/MultipleStopRounded";
 import UpdateRoundedIcon from "@mui/icons-material/UpdateRounded";
 import {grey} from "../themes/colors/aptosColorPalette";
 
+type Color =
```

Comment:
ochh I also had it a couple of days ago, are we allowing all colors? we can restrict to only the color types we use in the app. from looking at the code here it seems like we only use "primary"?